aten/src/ATen/native/LegacyDefinitions.cpp

```diff
@@ -12,817 +10,957 @@ void* data_ptr(const Tensor & self) {
   return self.unsafeGetTensorImpl()->slow_data();
 }
 
-Tensor & set_(Tensor& self, Storage source) {
-  return at::legacy::th::_th_set_(self, source);
+Tensor & set__cpu(Tensor& self, Storage source) {
+  return at::legacy::cpu::_th_set_(self, source);
```

Comment:
Dumb question: Why can't we just have the native functions call the legacy function directly. So for example, instead of `dispatch: CPU: set__cpu`, just say `dispatch: CPU: at::legacy::cpu::_th_set_`?

Reply:
@gchanan, maybe you know the answer to this question?

Reply:
I don't remember if the codegen supports that or not, but it seems like a reasonable thing to try.